doc/classes/RichTextLabel.xml

```diff
@@ -80,6 +80,13 @@
 				Returns the total number of paragraphs (newlines or [code]p[/code] tags in the tag stack's text tags). Considers wrapped text as one paragraph.
 			</description>
 		</method>
+		<method name="get_parsed_text" qualifiers="const">
+			<return type="String">
+			</return>
+			<description>
+				Returns the BBCode text without BBCode mark-up.
```

Comment:
I'd change the description to `Returns the text without BBCode mark-up.` or `Returns the text with BBCode mark-up removed.`